wremnants/datasets/datagroups.py

```diff
@@ -172,32 +184,69 @@ def loadHistsForDatagroups(
         nominalIfMissing=True, applySelection=True, forceNonzero=True, pseudodata=False,
         preOpMap={}, preOpArgs={}, scaleToNewLumi=-1
     ):
+        logger.debug("Calling loadHistsForDatagroups()")
         logger.debug(f"the basename and syst is: {baseName}, {syst}")
         logger.debug(f"The procsToRead and excludedProcs are: {procsToRead}, {excluded_procs}")
         if self.rtfile and self.combine:
             self.setHistsCombine(baseName, syst, channel, procsToRead, excluded_procs, label)
         else:
-            self.setHists(baseName, syst, procsToRead, label, nominalIfMissing, applySelection, forceNonzero, preOpMap, preOpArgs, scaleToNewLumi=scaleToNewLumi)
-
-    def getDatagroups(self, excluded_procs=[]):
+            self.setHists(baseName, syst, procsToRead, label, nominalIfMissing, applySelection,
+                          forceNonzero, preOpMap, preOpArgs,
+                          scaleToNewLumi=scaleToNewLumi, excludeProcs=excluded_procs)
+
+    def addGroups(self, dictToAdd, canReplaceKey=False):
```

Comment:
Based on how it is used I think it would make more sense if this was

addGroup(self, name, group_data):
    if name in self.groups.keys():
         logger.warning...
    self.groups[name] = group_name

Reply:
ok, yes, I can pass the name and the item separately, it was done like that to allow adding an actual dictionary, but maybe doing it item by item is cleaner, I will fix it